devel/README.md

```diff
@@ -33,10 +33,25 @@ cd image-builder-frontend/devel
 ./setup.sh
 ```
 
+## Environment Variables
+
+For the Image Builder backend to upload successfully, you need to set some environment variables. For AWS, for example, set the following environment varaibles in the `.env` file:
+
+```
+OSBUILD_AWS_REGION
+OSBUILD_AWS_ACCESS_KEY_ID
+OSBUILD_AWS_SECRET_ACCESS_KEY
+OSBUILD_AWS_S3_BUCKET
+```
+
+And then add to the list of environment variables for the backend container in the `docker-compose.yml` file.
+
+The config variables for the Image Builder backend can be found [here](https://github.com/osbuild/image-builder/blob/100af34e53403ef58009ce3de071c5104f1742d6/internal/config/config.go#L5-L30).
```

Comment:
Hm, why not just link to https://github.com/osbuild/image-builder/blob/main/internal/config/config.go ? As in future these might expand.

Reply:
I was undecided between the commit hash and the link to the main repo, just in case the location changed.
But that's a good point. I'll fix that now